mwdb/web/src/components/AttributesAddModal.js

```diff
@@ -178,19 +194,41 @@ export default function AttributesAddModal({ isOpen, onAdd, onRequestClose }) {
                             />
                         ) : (
                             <AceEditor
+                                placeholder={richExampleValue}
                                 mode="json"
                                 theme="github"
                                 wrapEnabled
                                 onChange={(input) => setAttributeValue(input)}
                                 value={attributeValue}
-                                width="300px"
+                                width="500px"
                                 height="150px"
                                 setOptions={{
                                     useWorker: false,
                                 }}
                             />
                         )}
                     </div>
+                    {richTemplate && attributeType === "object" ? (
```

Comment:
That preview should be also available for string type (see Simple URL)
```suggestion
                    {richTemplate ? (
```